### PR TITLE
842094 : Update PDF "Getting Started" topic with screenshots.

### PR DESCRIPTION
**Pull request by Sri-hari-haran-g**
Hi All,

I have added the Screenshot image for "Getting Started" topic. Kindly review this.

Regards,
Srihariharan Gopalakrishnan

### CODE REVIEW

File: File-Formats/PDF/Create-PDF-file-in-ASP-NET-Web-Forms.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ PdfPage page = doc.Pages.Add();
 //Create PDF graphics for the page
 PdfGraphics graphics = page.Graphics;
 //Load the image from the disk.
-PdfBitmap image = new PdfBitmap(Server.MapPath("~/Autumn Leaves.jpg"));
+PdfBitmap image = new PdfBitmap(Server.MapPath("~/MicrosoftTeams-image.png"));
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Create-PDF-file-in-ASP-NET-Web-Forms.md:94]: Change the image name

File: File-Formats/PDF/Create-PDF-file-in-UWP.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ PdfPage page = doc.Pages.Add();
 //Create PDF graphics for the page.
 PdfGraphics graphics = page.Graphics;
 //Load the image as stream.
-Stream imageStream = typeof(MainPage).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Data.Autumn Leaves.jpg");
+Stream imageStream = typeof(MainPage).GetTypeInfo().Assembly.GetManifestResourceStream("MicrosoftTeams-image.png");
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Create-PDF-file-in-UWP.md:144]: Change the image name

File: File-Formats/PDF/Create-PDF-file-in-Windows-Forms.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ PdfPage page = doc.Pages.Add();
 //Create PDF graphics for the page
 PdfGraphics graphics = page.Graphics;
 //Load the image from the disk.
-PdfBitmap image = new PdfBitmap("Autumn Leaves.jpg");
+PdfBitmap image = new PdfBitmap("MicrosoftTeams-image.png");
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Create-PDF-file-in-Windows-Forms.md:107]: Change the image name

File: File-Formats/PDF/Create-PDF-file-in-ASP-NET-Core.md [hunk 4]
```diff
@@ -100,7 +100,7 @@ PdfPage page = doc.Pages.Add();
 //Create PDF graphics for the page.
 PdfGraphics graphics = page.Graphics;
 //Load the image as stream.
-FileStream imageStream = new FileStream("Autumn Leaves.jpg", FileMode.Open, FileAccess.Read);
+FileStream imageStream = new FileStream("MicrosoftTeams-image.png", FileMode.Open, FileAccess.Read);
```
[the remark this answers is not in the document]
Comment (Sri-hari-haran-g) [reply File-Formats/PDF/Create-PDF-file-in-ASP-NET-Core.md]: Changed the image name to "Adventure Cycles.png".

**Review by praveenHari24 — APPROVED** [review]

**Pull request merged**